src/components/CardView.js

```diff
@@ -0,0 +1,171 @@
+        width: 340,
+        height: 322,
+        color: 'white',
+        float: 'left',
+        margin: 12,
+        borderRadius: 10
+    },
+    like: {
+        position: 'relative',
+        float: 'left',
+        top: 25,
+        left: 3,
+        width: 40,
+        height: 40,
+        boxShadow: '5px 5px 5px 0px rgba(0,0,0,0.75)'
+    },
+    media: {
+        width: 340,
+        height: 'calc(100% - 96px)',
+        backgroundSize: '100% 100%'
+    },
+    full_media: {
+        width: '100%',
+        height: 426,
+        backgroundSize: '100% 100%'
+    },
+    header: {
+        height: 64,
+        margin: 0,
+        maxHeight: 64,
+        overflowY: 'auto',
+        textJustify: 'inter-word',
+        wordBreak: 'break-word',
+        textAlign: "center",
+        paddingTop: 16,
+        paddingBottom: 16,
+        marginLeft: 1,
+        marginRight: 1
+    },
+    full_player: {
+        maxWidth: 640,
+        color: 'white',
+        margin: 12
+    },
+    full_text: {
+        color: 'white',
+        float: 'left',
+        textAlign: 'justify',
+        textJustify: 'inter-word',
+        wordBreak: 'break-word',
+        margin: 20,
+        fontSize: '20px'
+    },
+    back: {
+        color: 'white',
+        display: 'block',
+        textAlign: 'center',
+        marginBottom: 2
+    }
+}));
+
+export default function CardView(props) {
+    const classes = useStyles(),
+    [snack, setSnack] = useState({ message: "", severity: "" }),
+    [isDisposed, setDispose] = useState(false),
+    [isHiddenLike, setHideLike] = useState(false),
+    location = useLocation(),
+    isLocationHome = location.pathname === '/',
+    isLocationFavorites = location.pathname === '/favorites',
+    isLocationFavorite = location.pathname.includes('/favorite/'),
+    severityType = {
+        ERROR: 'error',
+        WARNING: 'warning',
+        INFO: 'info',
+        SUCCESS: 'success'
+    }
+
+    const saveToDB = async (data) => {
+        await axios.post(`${utils.FULL_URL}/image`, data);
+        setSnack({ message: "Selected data has been saved successfully!", severity: severityType.SUCCESS });
+    }
+
+    const removeFromDB = async (id) => {
+        await axios.delete(`${utils.FULL_URL}/image/${id}`);
+        setSnack({ message: "Selected data has been removed successfully!", severity: severityType.SUCCESS });
+    }
+
+    const handleLikes = (e) => {
+        if (isLocationFavorite || isLocationFavorites) {
+            //remove from DB
+            removeFromDB(props.data._id);
+            setDispose(true);
+        }
+        else {
+            //save to DB
+            saveToDB({ title: props.data.title, url: props.data.url, explanation: props.data.explanation ? props.data.explanation : ""});
+            setHideLike(true);
+        }
+    }
+
+    const handleFavClick = (e) => {
+        if (!isLocationFavorites)
+            e.preventDefault();
+    }
+
+    return (
+			<div>
+				{isDisposed ? null : (
```

Comment:
Instead of using ternary with `null` just use the `&&` operator with something truthy, like:
`!isDisposed && <div>....`

Here is a link to learn a little more --> https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator